src/app/components/List/List.module.css

```diff
@@ -0,0 +1,26 @@
+.listElement {
+  background-color: var(--primary-color);
```

Comment:
Primary-color seams a bit off for a background color. Consider to create another variable for background color.

Reply:
Thanks for the tip, I have generally reconsidered my naming in global.css.